corehq/pillows/app_submission_tracker.py

```diff
@@ -10,21 +10,23 @@
 from corehq.util.doc_processor.sql import SqlDocumentProvider
 from couchforms.models import XFormInstance, XFormArchived, XFormError, XFormDeprecated, \
     XFormDuplicate, SubmissionErrorLog
-from pillowtop.checkpoints.manager import PillowCheckpoint
+from pillowtop.checkpoints.manager import KafkaPillowCheckpoint
 from pillowtop.feed.interface import Change
 from pillowtop.pillow.interface import ConstructedPillow
 from pillowtop.processors.form import FormSubmissionMetadataTrackerProcessor
 from pillowtop.reindexer.reindexer import Reindexer
 
 
-def get_form_submission_metadata_tracker_pillow(pillow_id='FormSubmissionMetadataTrackerProcessor'):
+def get_form_submission_metadata_tracker_pillow(pillow_id='FormSubmissionMetadataTrackerProcessor', params=None):
     """
     This gets a pillow which iterates through all forms and marks the corresponding app
     as having submissions. This could be expanded to be more generic and include
     other processing that needs to happen on each form
     """
-    change_feed = KafkaChangeFeed(topics=[topics.FORM, topics.FORM_SQL], group_id='form-processsor')
-    checkpoint = PillowCheckpoint('form-submission-metadata-tracker', change_feed.sequence_format)
+    checkpoint_id = 'form-submission-metadata-tracker'
+    kafka_topics = [topics.FORM, topics.FORM_SQL]
```

Comment:
got quite confused as to whether these should be tuples or not. Would be nice to standardise from some point down.